_posts/2021-02-11-Creating-A-Game.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: Creating A Game
+description: 
+summary: 
+tags:
+---
+Today, I did typing club for about an hour. We also built a game, we had to create a new folder locally and work through a tutorial that Kajari provided for us we finished with a game. [This is the article we used](https://developer.mozilla.org/en-US/docs/Games/Tutorials/2D_Breakout_game_pure_JavaScript) After lunch, I met with Kajari and she provided some feedback. One thing that I need to pay more attention to is to small details, I think that I tend to get overwhelmed and I stress about finishing something, so I tend to rush things. I will be working on paying more attention to details and learning how to take things calmly.
```

Comment:
`JavaScript) After lunch`. <-- Make a new paragraph after the article link and put a period to close out that sentence. Then make a new paragraph starting with `After lunch`